### PR TITLE
Add endpoint for RecognizePlaces.

### PR DESCRIPTION
**Pull request by Spaceenter**
This is trying to migrate https://github.com/datacommonsorg/website/pull/2417/files# to Mixer.

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Thanks!

File: internal/server/handler.go [hunk 1]
```diff
@@ -297,3 +297,10 @@ func (s *Server) BulkFindEntities(
 ) (*pb.BulkFindEntitiesResponse, error) {
 	return recon.BulkFindEntities(ctx, in, s.store, s.mapsClient)
 }
+
+// RecognizePlaces implements API for Mixer.RecognizePlaces.
```
Comment (shifucun) [inline internal/server/handler.go:301]: put this in handler_v1.go since it servers /v1 API?

Comment (Spaceenter) [reply internal/server/handler.go]: done

File: proto/recon.proto [hunk 2]
```diff
@@ -127,4 +127,18 @@ message BulkFindEntitiesResponse {
...
+
+message RecognizePlacesRequest {
+  string query = 1;
+}
+
+message RecognizePlacesResponse {
+  message Item {
+    // Required.
+    string span = 1;
+    // Optional.
```
[the remark this answers is not in the document]
Comment (Spaceenter) [reply proto/recon.proto]: done

Made a message for Place too.

**Pull request merged**